units/system-update-pre.target

```diff
@@ -0,0 +1,16 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Offline System Update (Pre)
+Documentation=man:systemd.offline-updates(7)
+Documentation=man:systemd.special(7) man:systemd-system-update-generator(8)
+RefuseManualStart=yes
+After=sysinit.target
+Before=system-update.target
```

Comment:
Unfortunately this does not guarantee any ordering, because system-update-pre.target will be started in parallel with any .service units that are part of system-update.target, and can potentially "lose the race". So an ordering in *those* units to be After=system-update-pre.target is also necessary.

Those additional dependencies will have to be added to those units. Please update the systemd.offline-updates(7) man page too (point 4. in Recommnedations).